front/lib/api/assistant/feedback.ts

```diff
@@ -41,47 +118,58 @@ export async function getConversationFeedbacksForUser(
     return new Err(new ConversationError("conversation_access_restricted"));
   }
 
-  const messages = await Message.findAll({
+  const feedbackForMessages = await Message.findAll({
```

Comment:
This should live on the Resource. No Sequelize interactions outside of resources for new models :pray:

Reply:
Should live on AgentMessageFeedbackResource